pyproject.toml

```diff
@@ -109,3 +109,86 @@ source-includes = [
 [build-system]
 requires = ["pdm-backend"]
 build-backend = "pdm.backend"
+
+[tool.ruff.lint]
+# default rules include pyflakes ("F")
+# see https://docs.astral.sh/ruff/configuration/#using-pyprojecttoml
+# "F" contains autoflake, see https://github.com/astral-sh/ruff/issues/1647
+# add more rules
+extend-select = [
```

Comment:
I'd much rather have this be `select` and explicitly list everything. I don't like the possibility of `ruff` changing their defaults and we are suddenly running something we did not intent to.

Reply:
Changed.